.github/workflows/dotnet-format.yml

```diff
@@ -2,29 +2,22 @@ name: dotnet format
 
 on:
   push:
-    branches: [ main ]
-    paths:
-    - '**.cs'
-    - '.editorconfig'
```

Comment:
Why was this removed?

Reply:
I removed it because this action runs relatively fast and in parallel. If there is any change to the action, we can see if it fails.

Reply:
Fair enough, thanks.